packages/fast-usdc/src/exos/advancer.js

```diff
@@ -81,21 +82,45 @@ const AdvancerKitI = harden({
   }),
 });
 
+/**
+ * Address hooks only deal in strings, so see if we can parse
+ * chainId to an integer.
+ * @param {string} chainId
+ */
+const formatChainId = chainId => {
```

Comment:
why not just leave it as a string? there's no arithmetic being performed.

Reply:
For eip155 chains, chain ID is `uint`, so the thinking was to preserve that

Reply:
I don't know of a current use case, but could imagine a smart contract function signature that expects an integer for a chainId parameter. Perhaps these types of functions can handle this concern themselves instead of the orchestration api. 

For eth ecosystem chain ids, let's please at least use `{number}` for the type value.